content/applications/productivity/calendar/google.rst

```diff
@@ -153,3 +153,56 @@ Now, Odoo Calendar is successfully synced with Google Calendar!
 
    Events can be created in Google Calendar without sending a notification by selecting
    :guilabel:`Don't Send` when prompted to send invitation emails.
+
+Google OAuth FAQ
+================
+
+Production VS Testing Publishing Status
+---------------------------------------
+
+Choosing :guilabel:`Production` as the :guilabel:`Publishing Status` (instead of
+:guilabel:`Testing`) will display the following warning message:
+
+.. image:: google/published-status.png
+   :align: center
+   :alt: OAuth is Limited to 100 Sensitive Scope Logins.
+
+To correct this warning, navigate to the `Google API Platform
+<https://console.cloud.google.com/apis/credentials/consent>`_. If the :guilabel:`Publishing status`
+is :guilabel:`In Production`, click :guilabel:`Back to Testing` to correct the issue.
+
+No Test Users Added
+-------------------
+
+If no test users are added to the OAuth consent screen, then a 403 access denied error will
```

Comment:
Do you think "403 access denied error" should be bolded or something to emphasize it from the rest of the sentence? If it shouldn't be (for a technical reason) then disregard

Reply:
You give "Authorization Error" a gui label below - maybe this could get the same?